### ISSUE TITLE
Missing import statement in documentation (Get Started)

### ISSUE THREAD

**Issue by geetu040** [qdrant/qdrant-client#475]
In the documentation, first page of [Get Started](https://python-client.qdrant.tech), under Examples, the code is given as

Create a new collection
```python
client.recreate_collection(
   collection_name="my_collection",
   vectors_config=VectorParams(size=100, distance=Distance.COSINE),
)
```

It is not clear for the first-time users how to import the classes, instead it should look like this

```python
from qdrant_client.models import VectorParams, Distance

client.recreate_collection(
   collection_name="my_collection",
   vectors_config=VectorParams(size=100, distance=Distance.COSINE),
)
```

**Comment by shankariraja** [comment]
Thanks for reporting! I'll add the missing import statements as suggested.

**Comment by geetu040** [comment]
This pull request #464 already fixes this (reviewed but not yet merged)

**Comment by joein** [comment]
Merged #464 

Thanks all for the contributions
